Sources/Mantis/Helpers/TransformRecord.swift

```diff
@@ -0,0 +1,102 @@
+import UIKit
+
+public enum TransformType: Int {
+    case resetTransforms
+    case transform
+}
```

Comment:
Does this enum must be Int or it can be removed?

the case name `resetTransforms` has `s` but the second case `transform` does not have `s`, did you name them like this on purpose?

Reply:
Removed the Int. 

"resetTransorms" makes sense because it is undoing n-transforms from the stack